apps/i18n/common/es_es.json

```diff
@@ -1499,6 +1509,13 @@
   "standardsGetInvolvedDetails": "Aprenda cómo involucrar a sus [administradores]({adminLink}), [familiares]({parentLink}), y [docentes]({teacherLink}) en la educación de CS.",
   "standardsGetInvolvedDetailsForPrint": "Para aprender más sobre lo que tú puedes hacer en la educación de ciencias de la computadora como un [administrador] ({linkadmin}), [tutores]({Linktutores}), o [profesor]({Linkprofesor})",
   "standardsHowTo": "Como usar esta información",
+  "standardsHowToForPrint": "Cómo usar esta noticia",
+  "standardsHowToDetails": "Esta noticia enseña como Code.org [{NombreCurso}]({Linkcurso}) mapas según los [CSTA estándares]({linkcsta}) y como algunas lecciones en el{nombrecurso} esta clase ha sido completada. Completando todas las lecciones mediante las estándares con maestría no indica automáticamente -por favor usa tu mejor juicio para determinar como de bien los estudiantes de la clase o individuales han aprendido el material del curso. Como referencia, una lección de Code.org dura aproximádamente 45-65 minnutos de tiempo de estudio. Una lección \"atascada\" es considerada \"en proceso\" cuando al menos el 20% ha sido completado de la lección por el estudiante matriculado. Un atasco de la lección online es considerada completada cuando el estudiante matriculado ha terminado el 60% de la lección. Las lecciones sin atasco pueden ser marcadas como \"completas\" o \"incompletas\" por el profesor.",
```

Comment:
@castro-jorge looks like we're still getting variables translated in Spanish

Reply:
Thanks, I just let the new translator know about this.